plugins/modules/digital_ocean.py

```diff
@@ -204,7 +202,9 @@
     import dopy
     from dopy.manager import DoError, DoManager
 
-    if LooseVersion(dopy.__version__) >= LooseVersion("0.3.2"):
+    # NOTE: Expressing Python dependencies isn't really possible:
+    # https://github.com/ansible/ansible/issues/62733#issuecomment-537098744
+    if dopy.__version__ >= "0.3.2":  # Naive lexographical check
```

Comment:
If you want to keep a similar approach, you can use something [like this](https://packaging.pypa.io/en/latest/version.html) instead:
```
from packaging.version import Version

if Version(dopy.__version__) >= Version("0.3.2"):
```

Reply:
Ah yes, thanks for this suggestion -- pushed!